account_invoice_import_invoice2data/tests/test_invoice2data_template.py

```diff
@@ -0,0 +1,9 @@
+# -*- coding: utf-8 -*-
+# © 2017 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp.tests.common import TransactionCase
+
+
+class TestInvoice2dataTemplate(TransactionCase):
+    def test_invoice2data_template(self):
+        pass
```

Comment:
Completing this test would be nice.

Reply:
Tests are created, also the original test is loaded now.